VehicleCode/TestingAndCalibration/SensorTesting/src/BrakeSensor.cpp

```diff
@@ -0,0 +1,279 @@
+#include "BrakeSensor.h"
```

Comment:
Make sure you update all versions of all files changed. The "updates" script and accompanying "script_instructions.txt" in the "VehicleCode" directory can help with this